app/views/dashboard/dashboard.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="dashboard_container">
+  <h4>Jobs by level of interest</h4>
+  <table class="jobs_by_interest table">
+    <th>Interest Level</th><th>Number of Jobs</th>
+    <% @jobs_by_interest.each do |interest, count| %>
+      <tr>
+        <td><%= interest %></td>
+        <td><%= count %></td>
+      </tr>
+    <% end %>
+  </table>
+</div>
+
+<div class="dashboard_container">
+  <h4>Top Three Companies by Job Interest Level</h4>
+  <table class="top_three table">
+    <th>Company Name</th><th>Level of Interest</th>
+    <% @top_three.each do |company| %>
+      <tr>
+        <td><%= link_to company.name, company %></td>
+        <td><%= company.jobs.average(:level_of_interest).to_f %></td>
```

Comment:
extract the averaging out to method in your model so you can both reuse it and keep the view template clean!